src/peer/meshRoom.js

```diff
@@ -220,6 +227,15 @@ class MeshRoom extends Room {
 
     if (connection) {
       connection.handleCandidate(candidateMessage);
+    } else {
+      // Looks like PeerConnection hasn't completed setRemoteDescription
+      if (!this._queuedMessages[candidateMessage.connectionId]) {
```

Comment:
This condition becomes always false, probly.
I assume that we want to check if the array is not empty in this statement, but this won't work like that.  This should be
```javascript
 if (this._queuedMessages[candidateMessage.connectionId].length > 0) {
```

FYI, try below :)
```javascript
if (![]) {
 console.log("true!")
} else {
 console.log("false!")
}
```